src/core/Core.js

```diff
@@ -169,6 +169,8 @@ class Uppy {
         data: file.data,
         progress: {
           percentage: 0,
+          bytesUploaded: 0,
+          bytesTotal: file.data.size || 0,
```

Comment:
Should be fine from there on, since on progress events this happens:
https://github.com/transloadit/uppy/blob/master/src/plugins/Multipart.js#L48-L57